app/models/assignment.rb

```diff
@@ -137,7 +138,15 @@ def tips
   end
 
   def increment_attemps!
-    self.attemps_count += 1 unless passed?
+    self.attempts_count += 1 if failed? || errored?
```

Comment:
Why?

Reply:
because we don't want failed attempts to go up for aborted status

Reply:
I just noticed that this would mean attempts don't increment for passed_with_warnings either though :thinking: and this is probably not desirable behavior considering the html runner focuses mostly on expectations

Reply:
Exactly, that is the point!

Reply:
Done